instagram-app/src/App.js

```diff
@@ -1,35 +1,28 @@
 import React from 'react';
 import './App.css';
-import PostContainer from './components/PostContainer/PostContainer';
-import SearchBar from './components/SearchBar/SearchBar'
-
-import dummyData from './dummy-data';
-
+import PostsPage from './components/PostContainer/PostsPage'
+import withAuthenticate from './components/authentication/withAuthenticate';
+import Login from './components/Login/Login';
 
 
 
 class App extends React.Component {
-  constructor(){
+  constructor() {
     super();
-    this.state = {
-      dummy: []
-    };
-
-  }
-  componentDidMount(){
-    this.setState({dummy:dummyData});
+    this.state = {};
```

Comment:
No need to declare an empty `state` object. Also, since you don't seem to need any features from class components, you might be fine with just using a functional component here.